src/erlcloud_sns.erl

```diff
@@ -487,11 +489,11 @@ get_notification_attribute(Attribute, Notification) ->
 
 
 
--spec(set_topic_attributes/3 :: (sns_topic_attribute_name(), string(), string()) -> ok).
+-spec(set_topic_attributes/3 :: (sns_topic_attribute_name(), parameter(), parameter()) -> ok).
```

Comment:
Line 492 is too long: -spec(set_topic_attributes/3 :: (sns_topic_attribute_name(), parameter(), parameter()) -> ok)..